backends/mysql/README.md

```diff
@@ -0,0 +1,21 @@
+# Storage Backend - Mysql
+
+Grpc service which can read a write spans to a mysql cluster 
```

Comment:
It might be useful for newbies to mention that mysql still needs elasticsearch for indexing. When I first looked at your branch, I quickly looked for indexing..